### DISCUSSION TITLE
AWS Copilot nginx: [emerg] cannot load certificate "/etc/ssl/web/ssl.crt": BIO_new_file() failed (SSL: error:02001002:

### DISCUSSION THREAD

**Discussion by jericopulvera** [serversideup/docker-php#93] [Q&A] [answered]
I'm trying to deploy this via AWS Copilot and I'm having issue. 

The error is  ```nginx: [emerg] cannot load certificate "/etc/ssl/web/ssl.crt": BIO_new_file() failed (SSL: error:02001002:system library:fopen:No such file or directory:fopen('/etc/ssl/web/ssl.crt','r') error:2006D080:BIO routines:BIO_new_file:no such file)```

I just copied the Dockerfile from https://github.com/superfly/flyctl/tree/master/scanner/templates/laravel/.fly

I have tried adding ENV SSL_MODE=off on the Dockerfile as well unfortunately the issue still persist.

**Comment by jericopulvera** [comment, marked as answer]
I have figured it out. SSL_MODE option is only available in the latest version 2.0.2 and I'm using 1.5.0

**Reply by jericopulvera** [reply]
I happen to encounter this after deleting the octane conditional that contains this line.

 ```cp .fly/nginx-default /etc/nginx/sites-available/default;```
 
 This line is the one that bypasses the ssl warning making the fly template work as expected from the start.

**Reply by jaydrogers** [reply]
Thanks for sharing your solution!

I am going to ping @fideloper on this one as an FYI since he is the one who brought these images to Fly 👍

**Reply by fideloper** [reply]
👀👍
